util/test/test_blocktime.py

```diff
@@ -147,14 +153,63 @@ def test_getstfinBlocks():
     assert fin == 0
 
 
+@enforce_types
+def test_ethTimestamptoBlock():
+    networkutil.connect(1)
```

Comment:
Every unit test in this module automatically calls `setup_function()` before doing the body of the test. Inside `setup_function()`, it calls `networkutil.connect()`. 

Every unit test also auto-calls `teardown_function()` after doing the body of the test. Inside `teardown_function()`, it calls `networkutil.disconnect()`. 

Therefore:
- why does this test (`test_ethTimestamptoBlock`) have `networkutil.connect(1)` in the first line? (If it does need it, it should pass in `networkutil.DEV_CHAINID` not `1`.
- and, why does it again have `networkutil.connect(networkutil.DEV_CHAINID)` in the last line?